components/Pages/Rewards/tokens.ts

```diff
@@ -0,0 +1,44 @@
+const TOKENS = [
+  {
+    title: "Participation Rewards",
+    contract: { address: process.env.NEXT_PUBLIC_PARTICIPATION_REWARDS_CONTRACT_ADDRESS },
```

Comment:
instead of contract and then address why not just rename the key `contractAddress`?